### ISSUE TITLE
Luminosity contrast ratio of keyboard focus indicator on Authorize button fails to meet the required ratio of 3:1: A11y_VS Code extension for API Center_Generate HTTP file_Authorize_Non Text Contrast.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode#237882]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG1.4.11; #BM_VisualStudioCodeClient_Win32_JULY2024; #DesktopApp; #Visual Studio Code Client; #A11yMAS; #NonText Contrast; #A11YWCAG2.1; 

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Generate HTTP file" using the down arrow and press enter.
7. An HTTP file will appear.
8. Tab till Authorize button.
9. Verify whether the Luminosity contrast ratio of keyboard focus indicator on Authorize button meet the required ratio of 3:1 or not.

### Actual Result:
Luminosity contrast ratio of keyboard focus indicator on Authorize button fails to meet the required ratio of 3:1
Note: This issue is observed throughout the feature.

### Expected Result:
Luminosity contrast ratio of visual focus indicator on authorize button should be greater than or equal to required ratio of 3:1.

### User Impact:
People with low vision often have difficulty perceiving tab focus that have insufficient contrast if Luminosity contrast ratio of visual focus indicator fails to meet the required ratio of 3:1. This can be exacerbated if the person has a color vision deficiency that lowers the contrast even further. Providing a lightness difference of 3:1 or greater can make these items more distinguishable when the person does not see a full range of colors

### Attachment:

![Image](https://github.com/user-attachments/assets/b9a042a6-3318-40cb-86e0-b74d7f1d772d)

**Comment by meganrogge** [comment]
What theme are you using?

**Comment by AnKushSingh05** [comment]
Hi @meganrogge we are using the visual studio light theme.

<img width="800" height="450" alt="Image" src="https://github.com/user-attachments/assets/f02fc93a-a3b6-4fe8-a339-a433fe2b93e9" />

**Comment by hawkticehurst** [comment]
Hi @AnKushSingh05, based on the screenshots, it seems like this issue is only showing up in the Azure API Center extension? Can you confirm?

If that's the case, both the yellow outline color and lack of dark theme adherance would be caused by implementation details in the extension itself and you would need to file this bug in the [extension repo](https://github.com/microsoft/vscode-azureapicenter). Even more specifically, your screenshots seem to show that the extension is using a webview (aka a sandboxed webpage inside VS Code) to render content –– in these scenarios it's up to the extension author to implement the correct support for VS Code themes.
